packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4997,6 +5001,8 @@ class Semantics extends SingleChildRenderObjectWidget {
   ///
   /// This setting is often used in combination with [SemanticsConfiguration.isSemanticBoundary]
   /// to create semantic boundaries that are either writable or not for children.
+  /// 
+  /// Setting [properties.route] to true will force this value to be true as well.
```

Comment:
Where is this enforced? 

Should we maybe say something along the lines: If you set a `route` to true this also has to be true and then have an assert enforcing that Instead of magically setting it?